styles/style.css

```diff
@@ -136,55 +136,7 @@ body {
   font-family: helvetica, sans-serif;
```

Comment:
I might put a git ignore on the .css and the .cssmap, they don't really need to be in the repo and they make these PR's a little confusing hahaha